[DOCS] Update Compare Two Tables with the Onboarding Data Assistant (#8331)
great-expectations/great_expectations: docs/docusaurus/docs/guides/expectations/advanced/how_to_compare_two_tables_with_the_onboarding_data_assistant.md

<details>
<summary><code>exclude_column_names</code>?</summary>

> Above, we excluded several cloumns, including <code>"VendorID",</code>, so Expectations won't be set against these columns.

Author: ```suggestion
In the previous example, specific columns were excluded to prevent Expectations from being set against them.
```

> Some dialects of SQL handle data types in different ways, which can lead to (among other things) mismatches in precision on some numbers.

Author: ```suggestion
Some SQL dialects handle data types in different ways, and this can cause precision mismatches on some numbers.
```

> For our hypothetical use case, we're willing to tolerate those slight inconsistencies, and so aren't setting expectations against the columns likely to generate those errors.

Author: ```suggestion
In our hypothetical use case these inconsistencies are tolerated, and therefore Expectations are not set against the columns likely to generate the errors.
```

> This is one example of the ways in which we can customize the Suite built by our Data Assistant.

Author: ```suggestion
This is an example of how an Expectation Suite created by the Data Assistant can be customized.
```
